packages/firebase_ml/android/src/main/java/io/flutter/plugins/firebaseml/FirebaseMLPlugin.java

```diff
@@ -0,0 +1,56 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.package io.flutter.plugins.firebaseml;
+
+package io.flutter.plugins.firebaseml;
+
+import androidx.annotation.NonNull;
+import io.flutter.embedding.engine.plugins.FlutterPlugin;
+import io.flutter.plugin.common.MethodCall;
+import io.flutter.plugin.common.MethodChannel;
+import io.flutter.plugin.common.MethodChannel.MethodCallHandler;
+import io.flutter.plugin.common.MethodChannel.Result;
+import io.flutter.plugin.common.PluginRegistry.Registrar;
+
+/** FirebaseMLPlugin */
+public class FirebaseMLPlugin implements FlutterPlugin, MethodCallHandler {
+
+  private static final String CHANNEL_NAME = "plugins.flutter.io/firebase_ml";
+
+  private MethodChannel channel;
+
+  @Override
+  public void onAttachedToEngine(@NonNull FlutterPluginBinding flutterPluginBinding) {
+    channel = new MethodChannel(flutterPluginBinding.getBinaryMessenger(), CHANNEL_NAME);
+    channel.setMethodCallHandler(this);
+  }
+
+  public static void registerWith(Registrar registrar) {
+    final MethodChannel channel = new MethodChannel(registrar.messenger(), CHANNEL_NAME);
+    channel.setMethodCallHandler(new FirebaseMLPlugin());
+  }
```

Comment:
optional nit: Usually visible static methods would go above instance methods in a class.

go/java-practices/class-structure#general

Reply:
I can't see the link, but note taken. Thank you. Done